packages/apps/human-app/server/src/modules/register-address/register-address-profile.service.ts

```diff
@@ -7,7 +7,10 @@ import {
   SnakeCaseNamingConvention,
 } from '@automapper/core';
 import { AutomapperProfile, InjectMapper } from '@automapper/nestjs';
-import { RegisterAddressCommand, RegisterAddressDto } from './model/register-address.model';
+import {
+  RegisterAddressCommand,
+  RegisterAddressDto,
+} from './model/register-address.model';
 
 @Injectable()
 export class RegisterAddressProfile extends AutomapperProfile {
```

Comment:
It looks like RegisterAddressProfile is not necessary now.

Reply:
It is necessary

Reply:
Let's leave it for consistency and it might be useful for future expansion.

Reply:
It is used in actual implementation, it's not the matter of consistency